pkg/networkservice/mechanisms/kernel/server.go

```diff
@@ -0,0 +1,11 @@
+package kernel
+
+import (
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+	"github.com/networkservicemesh/sdk-kernel/pkg/networkservice/mechanisms/kernel/hostdevice"
+)
+
+func NewServer() networkservice.NetworkServiceServer {
+	// TODO
+	return hostdevice.NewServer()
```

Comment:
Question... do you plan on making a 'creation time' decision here on implementations?

Since whether a device is a PCI device or not is a runtime consideration... it might make sense to return:

```golang
func NewServer() networkservice.NetworkServiceServer {
    return chain.NewNetworkServiceServer(
        hostdevice.NewServer(),
        vethpair.NewServer(), // sdk-kernel's vethpair in this case, not the one from sdk-vppagent
   ) 
}
```

And have hostdevice.{Request,Close}() simply fall through if PCIDevice isn't set rather than returning an error.

Please note... lots of ways to do this, so feel free to explore and pick the one you think works best :)